.github/workflows/testPublish.yml

```diff
@@ -25,6 +25,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: npm ci
       - run: npm test
+      - run: npm run build:markdownDocs
```

Comment:
This is in the wrong spot. There is no need to run this script with the test scenario. It should be added to the `release:` scenario after `- run: npm ci`

Reply:
If we are running this in the cloud, the asset(s) that MM is creating need to be committed back to the repo. See the following code in the package.json to be updated. 

```
        {
          "assets": [
            "./CHANGELOG.md",
            "package.json"
          ]
        }
```

Reply:
Can I just add "./**/*.md" and have that cover it?

Reply:
I've added these changes but I am not sure how to test it. Wouldn't this only get run when we do a release?